src/types/Person.ts

```diff
@@ -9,3 +9,20 @@ export interface Person {
   mother?: Person,
   father?: Person,
 }
+
+export enum ErrorOption {
+  noError,
+  Wrong,
+  Empty,
+}
+
+export enum MagicWords {
+  FEMALE = 'f',
+  MALE = 'm',
+}
```

Comment:
why did you created this ?
you have already Sex enum for that

- MagicWords is a very bad name for the enum